home.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: frontpage
+title: Home
+permalink: /
+---
+
+## The goal of the project is to document repeatable processes, develop reusable tooling, and curate resources to support any effort in which a common type of data is collected or exchanged across disparate organizational boundaries.
```

Comment:
This seems like a long sentence to use for a header level 2, the text is quite large on the page. Wondering if we couldn't just make this plain text that would reside under the next section header (i.e., below line 9).

Reply:
This isn't a show stopper, though. Totally fine with this if we deem the proposed change not needed.

Reply:
@mheadd Totally fair. Updated it so it goes right into "Getting good reliable data is hard..." but added note about reusable tools and repeatable processes to banner.